dapps/marketplace/src/pages/transaction/_Progress.js

```diff
@@ -96,177 +112,133 @@ const AcceptOrReject = ({ offer, refetch, loading }) => (
+      <div className="d-flex flex-column mr-md-auto">
         <FinalizeOffer
           offer={offer}
           refetch={refetch}
           from={offer.listing.seller.id}
           className="btn btn-primary"
         >
-          <fbt desc="Progress.completeSale">Complete Sale</fbt>
+          <fbt desc="Progress.collectFunds">Collect Funds</fbt>
         </FinalizeOffer>
       </div>
     </div>
-    <Stages offer={offer} />
   </div>
 )
 
 const MessageSeller = ({ offer, refetch, loading, party }) => (
   <div className={`transaction-progress${loading ? ' loading' : ''}`}>
     <div className="top">
       <h4>
-        <fbt desc="Progress.nextSteo">Next Step</fbt>
-      </h4>
-      <div className="next-step">
         <fbt desc="Progress.giveShipping">
           Give your shipping address to seller
         </fbt>
-      </div>
-      <div className="help">
-        <fbt desc="Progress.clickButton">
-          Click the link below to open messaging
+      </h4>
+      <Stages className="mt-4" mini="true" offer={offer} />
+      <div className="mt-4">
+        <fbt desc="Progress.giveShippingAddress">
+          Make sure the seller knows where to send your item.
         </fbt>
+        <SendMessage to={offer.listing.seller.id} className="btn btn-link ml-2">
+          <fbt desc="Progress.messageSeller">Message Seller</fbt>
+        </SendMessage>
+      </div>
+      <div className="mr-auto">
+        <WithdrawOffer offer={offer} refetch={refetch} from={party} />
       </div>
-      <SendMessage to={offer.listing.seller.id} className="btn btn-link">
-        <fbt desc="Progress.messageSeller">Message Seller</fbt> &rsaquo;
-      </SendMessage>
-      <WithdrawOffer offer={offer} refetch={refetch} from={party} />
     </div>
-    <Stages offer={offer} />
   </div>
 )
 
 const WaitForSeller = ({ offer, refetch, loading, party }) => (
   <div className={`transaction-progress${loading ? ' loading' : ''}`}>
     <div className="top">
       <h4>
-        <fbt desc="Progress.nextStep">Next Step</fbt>
+        <fbt desc="Progress.waitForSeller">Wait for seller</fbt>
       </h4>
-      <div className="next-step">Wait for seller</div>
+      <Stages className="my-4" mini="true" offer={offer} />
       <div className="help">
         <fbt desc="Progress.sellerWillReview">
           The seller will review your booking
         </fbt>
       </div>
-      <WithdrawOffer offer={offer} refetch={refetch} from={party} />
+      <WithdrawOffer
+        className="mr-auto"
+        offer={offer}
+        refetch={refetch}
+        from={party}
+      />
     </div>
-    <Stages offer={offer} />
   </div>
 )
 
-const OfferWithdrawn = ({ offer, party, loading }) => (
+const OfferWithdrawn = ({ offer, viewedBy, loading }) => (
   <div className={`transaction-progress${loading ? ' loading' : ''}`}>
     <div className="top">
       <h4>
-        <fbt desc="Progress.offerWithdrawn">Offer Withdrawn</fbt>
+        {viewedBy === 'seller'
+          ? fbt(
```

Comment:
can we use a regular `<fbt>` component unless we really need the function version?

Reply:
Yeah this bit is pretty weird. I used the function because the linter would complain about this code: 
```
<fbt dec="smth">You've ....</fbt>
```
with the following error: 
```
origin/dapps/marketplace/src/pages/transaction/_Progress.js
  198:18  error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities
```

This way there were no linter problems. I kind of prefer calling a function over escaping the `'` character with `&apos;`

Do you know a better solution for this maybe?